pkg/sox

```diff
@@ -5,17 +5,22 @@ http://download.sourceforge.net/project/sox/sox/14.4.1/sox-14.4.1.tar.gz
 filesize=1111653
 sha512=a35c0f424ffaa51d06291e69b28ae7f2248a750fa8aded911dcd7e123877ecb5142c292f943eb723142c50dbd890b58163c6ef24c9c0573377855fad1b36fb25
 
-[deps]
+[deps.host]
+bash
+file
```

Comment:
some packages need file as a build dependency since it provides libmagic.a, which makes me think that it may make sense to provide a metapackage for that. anyway, if that is the case, file needs to go to [deps]